contracts/util/NumbersList.sol

```diff
@@ -6,7 +6,7 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 library NumbersList {
     using SafeMath for uint256;
 
-    uint256 private constant PERCENTAGE_TO_DECIMAL = 10000;
+    uint256 private constant PERCENTAGE_TO_DECIMAL = 100;
```

Comment:
This should be `10000`. The percentage `tolerance` has 2 decimal places added to it. So if we want tolerance of 1.25%, `tolerance = 125` and the decimal we want is `0.0125` so you have to divide by `10000`